packages/renderer/src/lib/preferences/PreferencesProxiesRendering.svelte

```diff
@@ -73,7 +73,7 @@ function validate(event: any) {
       </select>
     </label>
 
-    {#if proxySettings}
+    {#if proxyState !== ProxyState.PROXY_SYSTEM}
```

Comment:
if I have selected `manual` then switch back to system, I will not see the update button, therefore I will not be able to update the settings ?